script.service.checkpreviousepisode/addon.xml

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.service.checkpreviousepisode" name="XBMC Check Previous Episode" version="0.3.1" provider-name="Lucleonhart">
+<addon id="script.service.checkpreviousepisode" name="XBMC Check Previous Episode" version="0.3.2" provider-name="Lucleonhart, bossanova808">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
 	<import addon="xbmc.addon" version="12.0.0"/>
   </requires>
   <extension point="xbmc.service" library="default.py" start="login"></extension>
   <extension point="xbmc.addon.metadata">
-    <summary>XBMC check previous episode state</summary>
+    <summary lang="en">XBMC check previous episode state</summary>
+    <summary lang="bg">Изгледан ли е предходния епизод</summary>
     <description lang="en">This scripts checks the previous episode from the one you started to watch and warns you if it is missing from your library or unwatched!</description>
     <description lang="de">Dieses Script überprüft die vorige Episode der TV Show die gerade gestartet werden soll und warnt falls diese noch nicht gesehen wurde oder nicht vorhanden ist!</description>
+    <description lang="bg">Когато пускате епизод от даден сериал скриптът проверява предходния и предупреждава ако той отсъства от библиотеката или не е изгледан.</description>
     <platform>all</platform>
-    <language></language>
-    
+    <language>en</language>
```

Comment:
As this script doesn't provide any content, either remove the tag or leave it empty.
see https://kodi.wiki/view/Addon.xml#.3Clanguage.3E